conbench/app/auth.py

```diff
@@ -106,6 +108,11 @@ def post(self):
 
                 return self.redirect("app.index")
             else:
+                log.info(
+                    "login request failed, response details: %s, %s",
+                    response,
+                    response.text,
+                )
```

Comment:
This does otherwise swallow useful error information (e.g., if the API response has status code 400 and details in the body about how the request was bad -- then this would be swallowed).